src/Spectre.libPeakFinder.Tests/PeakQualityCalculatorTest.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+* PeakQualityCalculatorTest.cpp
+* Tests for PeakQualityCalculator class.
+*
+Copyright 2018 Michal Gallus
```

Comment:
Thanks for credits

Reply:
![](http://i0.kym-cdn.com/entries/icons/facebook/000/000/246/copypasta.jpg)